runtime/vm/JFRChunkWriter.hpp

```diff
@@ -300,6 +319,14 @@ class VM_JFRChunkWriter {
 
 			writeJVMInformationEvent();
 
+			writePhysicalMemoryEvent();
+
+			writeCPUInformationEvent();
+
+			writeVirtualizationInformationEvent();
+
+			writeOSInformationEvent();
+
 			writeJFRHeader();
 
 			writeJFRChunkToFile();
```

Comment:
Make sure there are no errors before we write to file. Also add a tracepoint to indicate if there was an error